modules/manage/pages/cluster-maintenance/partition-recovery.adoc

```diff
@@ -0,0 +1,86 @@
+= Forced partition recovery
+:description: Recover a single partition using the Admin API.
+
+You can use the Redpanda Admin API to recover a partition that is unavailable and has lost a majority of its replicas. This can occur when the partition replicas have lost glossterm:Raft[] consensus, for instance if brokers in a Raft group fail, preventing the group from reaching a majority and electing a new leader. 
+
+Redpanda performs forced partition recovery by promoting the best available replica to leader, allowing the partition to become available for produce and consume. Typically, Redpanda chooses the replica with the highest offset. 
+
+CAUTION: Forced partition recovery allows some potential data loss on the partition if the best available replica is out of sync, ending up in an unclean leader election. Use this operation with caution, and only when brokers have failed beyond recovery to the point that the remaining replicas cannot form a majority. 
+
+NOTE: If you want to instead force recover all partitions in bulk from a set of failed brokers, use xref:./nodewise-partition-recovery.adoc[nodewise recovery].
+
+== Use the Admin API to recover a partition
+
+The following examples are based on the assumption that partition 0 in topic `test` is unavailable and its replicas cannot form a majority to elect a leader.
+
+. Call the xref:api:ROOT:admin-api.adoc#tag/Partitions/operation/get_topic_partitions[`/partitions/kafka/<topic>`] endpoint to determine the current broker and shard assignments of the partition replicas.
```

Comment:
The example here is good, I'm just checking whether there's also an RPK command to do this. If so, let's reference it in this section as well.

Reply:
I don't think so. @ztlpn ?